File: android/app/src/main/java/com/wanjuanlink/lychee/crossplatform/service/CrossPlatformServiceRegister.java
package com.wanjuanlink.lychee.crossplatform.service;

import com.wanjuanlink.lychee.crossplatform.handlers.*;

public class CrossPlatformServiceRegister {
    public static void register(){
        CrossPlatformService.register();
        CrossPlatformServiceMessageToNative.register();
    }
}

File: lib/common/xservice/cross-platform/service/CrossPlatformServiceRegister.dart

 import 'CrossPlatformService.dart';
 import '../handlers/CrossPlatformServiceMessageToFlutter.dart'; 
 
 class CrossPlatformServiceRegister{
 
  static register(){
    CrossPlatformService.regsiter();
    CrossPlatformServiceMessageToFlutter.regsiter();
   }

 }